.github/workflows/tests.yml

```diff
@@ -1,5 +1,5 @@
 name: Testing
-on: push
```

Comment:
I've already made this change in the main branch. Could you please rebase against it?